config-symfony5.4/services.yaml

```
@@ -96,3 +148,13 @@ when@dev:
                     id: 'App\DataCollector\MSPDataCollector'
                     template: 'data_collector/msp.html.twig'
                 - 'controller.service_arguments'
+
+when@test:
+    services:
+        # make classes in src/DataFixtures available to be used as services
+        # and have a tag that allows actions to type-hint services
+        App\DataFixtures\:
+            resource: '../fixtures/'
+            tags:
+                - 'doctrine.fixture.orm'
+                - 'kernel.event_subscriber'
```

github wil een newline
```suggestion
                - 'kernel.event_subscriber'

```